src/openforms/conf/base.py

```diff
@@ -794,3 +794,6 @@
 OPENFORMS_LOCATION_CLIENT = config(
     "OPENFORMS_LOCATION_CLIENT", "openforms.contrib.bag.client.BAGClient"
 )
+
+
+ADMIN_DISPLAY_DROP_DOWN_MENU_CONDITION_FUNCTION = "openforms.utils.django-two-factors.should_display_dropdown_menu"
```

Comment:
Please group this setting with the other admin index setting(s).

Also, in admin index, the prefix should be `ADMIN_INDEX` and not just `ADMIN`, so please update that PR as well.

Reply:
`django-two-factors` is not a valid python identifier -> use underscores

And rather put this in `src/openforms/utils/django_two_factor_auth.py`

Reply:
done